packages/fxa-content-server/app/styles/modules/_settings-totp.scss

```diff
@@ -115,7 +115,7 @@
   }
 }
 
-.replace-codes-link {
+.replace-codes-link, button.link:hover {
```

Comment:
Instead of `button.link:hover`, lets call it `.replace-codes-link:hover` and put it in a new line.